### ISSUE TITLE
Preconfigure a database connection

### ISSUE THREAD

**Issue by cafischer** [dbeaver/cloudbeaver#1608]
**Describe the bug**
In https://github.com/dbeaver/cloudbeaver/wiki/Connection-configuration it is described that database configuration has to be put into ${WORKSPACE}/GlobalConfiguration/.dbeaver/data-sources.json. However, the database connection did not show up in the admin interface. When I  instead put it into /opt/cloudbeaver/conf/initial-data-sources.conf the connection showed up as wished. Can you update the documentation or fix the problem with the proposed approach? 

**To Reproduce**
docker-compose.yml
```
  dbeaver:
    build: ./docker/dbeaver
    container_name: dbeaver
    volumes:
      - database_volume:/db
    ports:
      - 8080:8978
    environment:
      CB_SERVER_NAME: CloudBeaver CE Server
      CB_SERVER_URL: http://localhost:8080
      CB_ADMIN_NAME: someadmin
      CB_ADMIN_PASSWORD: something

volumes:
  database_volume:
```

Dockerfile 
```
FROM dbeaver/cloudbeaver:23.0.2

# not working:
ADD config/initial-data-sources.conf /opt/cloudbeaver/GlobalConfiguration/.dbeaver/data-sources.json  
# working:
# ADD config/initial-data-sources.conf /opt/cloudbeaver/conf/initial-data-sources.conf
```

initial-data-sources.conf
```
{
  "folders": {},
  "connections": {
    "postgresql-template-1": {
      "provider": "postgresql",
      "driver": "postgres-jdbc",
      "name": "PostgreSQL (Template)",
      "save-password": false,
      "show-system-objects": false,
      "read-only": true,
      "template": true,
      "configuration": {
        "host": "localhost",
        "port": "5432",
        "database": "postgres",
        "url": "jdbc:postgresql://localhost:5432/postgres",
        "type": "dev",
        "provider-properties": {
          "@dbeaver-show-non-default-db@": "false"
        }
      }
    },
    "sqlite-database": {
      "provider": "sqlite",
      "driver": "sqlite_jdbc",
      "name": "SQLite@localhost",
      "configuration": {
        "database": "/db/db.sqlite3",
        "url": "jdbc:sqlite:/db/db.sqlite3",
        "configurationType": "MANUAL",
        "type": "dev"
      }
    }
  },
  "connection-types": {
    "dev": {
      "name": "Development",
      "color": "255,255,255",
      "description": "Regular development database",
      "auto-commit": true,
      "confirm-execute": false,
      "confirm-data-change": false,
      "auto-close-transactions": false
    }
  }
}
```
When starting the application the sqlite database can not be seen in the admin interface under connections.

**Comment by DanHartman** [comment]
FWIW you can see the reason your `ADD` statement is being overwritten in the `ENTRYPOINT` script:

```sh
± docker image inspect dbeaver/cloudbeaver:23.0.2 | jq '.[0].Config.Entrypoint'
[
  "./run-server.sh"
]
```

```sh
± docker run --rm --entrypoint '' dbeaver/cloudbeaver:23.0.2 grep -- '-d' run-server.sh
[ ! -d "workspace/.metadata" ] && mkdir -p workspace/.metadata && mkdir -p workspace/GlobalConfiguration/.dbeaver && cp conf/initial-data-sources.conf workspace/GlobalConfiguration/.dbeaver/data-sources.json
```

I agree, the doc you linked to doesn't seem to acknowledge this startup process

**Comment by cafischer** [comment]
How would you add the data-sources.json config then?

**Comment by DanHartman** [comment]
I think you solved it already.  If you're "baking" that info in, put it in the `initial-data-sources.conf` file

**Comment by marcellodesales** [comment]
Just use a Volume instead of building a new docker image!

**Comment by TatyanaSsau** [comment]
Thank you for your suggestion, we are going to update the wiki article as soon as possible.

**Comment by dariamarutkina** [comment]
Fixed in 23.1.1
